.github/workflows/deploy_edge.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v2
       with:
-        node-version: ${{ matrix.node }}
+        node-version: ${{ matrix.nodet }}
```

Comment:
Can you explain what this change is doing?

Reply:
this might be a mistake, I will fix this